docs/ops/deployment/yarn_setup.md

```diff
@@ -262,6 +262,12 @@ Flink's YARN client has the following configuration parameters to control how to
 
 - `yarn.application-attempts`: The number of ApplicationMaster (+ its TaskManager containers) attempts. If this value is set to 1 (default), the entire YARN session will fail when the Application master fails. Higher values specify the number of restarts of the ApplicationMaster by YARN.
 
+## Setup for application priority on YARN
+
+Flink's YARN client has the following configuration parameters to setup application priority. These parameters can be set either from the `conf/flink-conf.yaml` or when starting the YARN session, using `-D` parameters.
+
+- `yarn.application.priority`: A non-negative integer indicating the priority for submitting a Flink YARN application. It will only take effect if YARN priority scheduling setting is enabled. Larger integer corresponds with higher priority. If priority is negative or set to '-1'(default), Flink will unset yarn priority setting and use cluster default priority, see [YARN Capacity Scheduling Doc](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html) for more info.
```

Comment:
I think we should use specific version that first supported this instead of `current` in the URL link.

Reply:
Thanks for the discussion on the JIRA issue! I will modify the doc description to specify the current supported version of YARN. And since most of our current jobs are still running with flink 1.5 API with priority feature we add, so the API calls 'appContext.setPriority' should be available in later versions.

Reply:
@walterddr Thanks for your review~ I have added the supported Hadoop version info according to the advice.